bundles/io/org.eclipse.smarthome.io.transport.bluetooth/src/main/java/org/eclipse/smarthome/io/transport/bluetooth/BluetoothAdapter.java

```diff
@@ -0,0 +1,245 @@
+/**
+ * Copyright (c) 1997, 2015 by Huawei Technologies Co., Ltd. and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.smarthome.io.transport.bluetooth;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+
+import org.eclipse.smarthome.io.transport.bluetooth.internal.events.BluetoothEvent;
+import org.eclipse.smarthome.io.transport.bluetooth.le.BluetoothLeScanner;
+
+/**
+ * BluetoothAdapter represents the physical Bluetooth adapter on the host machine (ex: hci0).
```

Comment:
Looks as if you have copied code from Kura. Either don't copy (but write it yourself), or make sure to refer to the original authors here as well (we would probably still need either their ok or a CQ for it).